_pages/training-development/working-groups-and-guilds-101.md

```diff
@@ -0,0 +1,205 @@
+          <a href="https://gsa-tts.slack.com/messages/g-devops">#g-devops</a>
+        </td>
+      </tr>
+      <tr>
+        <th id="diversity">Diversity</th>
+        <td class="col-description">
+          We help make TTS a great place to work for people of all backgrounds. <br />
+          <a href="https://gsa-tts.slack.com/messages/g-diversity">#g-diversity</a>
+        </td>
+      </tr>
+      <tr>
+        <th id="engineering">Engineering Practices</th>
+        <td class="col-description">
+          Promoting smart and scalable engineering practices across the frontend and backend. <br />
+          <a href="https://github.com/18F/development-guide">Homepage</a> &bull; <a href="https://gsa-tts.slack.com/messages/C02CD5VUQ">#dev</a>
+        </td>
+      </tr>
+      <tr>
+        <th id="security">Security &amp; Compliance</th>
+        <td class="col-description">
+          Our mission is to reduce the security and compliance barrier for all Federal and State government projects. <br />
+          <a href="https://gsa-tts.slack.com/messages/g-security-compliance">#g-security-compliance</a>
+        </td>
+      </tr>
+    </tbody>
+  </table>
+</div>
+
+### Guild leadership
+
+Guilds should be led people from different parts of TTS. They should expect to spend a few hours per week on guild leadership, meetings, and preparation. Individual guilds can set their own leadership terms, with the most common being one year. 
+
+Guild leadership is recognized by a staff member’s supervisor as a responsibility and they should be reviewed on their performance. However, it is an informal role. There are no administrative supervisory duties attached and there is no GS-level or previous leadership requirement.
+
+Typically guilds follow a lightweight leadership selection process:
+
+1. The guild asks for nominations via email and Slack posting. Guilds can choose to allow self-nomination only, or accept nominations by other people with confirmation of interest by the nominee. Nominees should have the verbal approval of their supervisor. Written nominations totalling no more than 400 words are recommended; guilds choose nomination questions.
+1. A panel of one-three people made up of guild leadership, other guild members and/or leadership from other guilds conduct brief (no more than 30 minute) interviews of candidates and makes a selection.
+1. A current guild leader announces the new leader, who takes up the position immediately.
+
+## Communities of Practice
+A Community of Practice (CoP) provides for government-wide knowledge-sharing. They attract members from across government agencies, and provide a good opportunity for Guilds to connect with others in government and influence best practices beyond TTS.
+
+Digital.gov [hosts the Communities of Practice](http://digital.gov/communities/), including the following:
+
+{% include dg-communities.html %}
+
+
+## Listservs, Google Groups and mailing lists
```

Comment:
There are a lot of listservs mentioned under this heading that are already getting pulled into the page via the Digital.gov API `{% include dg-communities.html %}`. You might consider removing those (or ditching the API), and just pointing to Digital.gov.

Reply:
Thanks! I had brought them over basically with no editing. Hadn't occurred to me to cross-reference. Will do.